packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -150,7 +150,7 @@ export function Checkbox({
                   onFocus={onFocus}
                 />
                 <span className={styles.checkBox}>
-                  <Icon name={iconName} size="small" color="white" />
+                  <Icon name={iconName} color="surface" />
```

Comment:
so that checkbox doesn't show a white checkmark in dark mode

(this might also merit just hiding the checkmark altogether when unselected, but this change has less... surface... area)